third_party/rust/adblock/v0_10/BUILD.gn

```diff
@@ -50,6 +50,7 @@ cargo_crate("lib") {
   cargo_pkg_authors = "Anton Lazarev <alazarev@brave.com>, Andrius Aucinas"
   cargo_pkg_name = "adblock"
   cargo_pkg_description = "Native Rust module for Adblock Plus syntax (e.g. EasyList, EasyPrivacy) filter parsing and matching."
+  cargo_pkg_repository = "https://github.com/brave/adblock-rust/"
```

Comment:
we can remove this from `config.js` now:
https://github.com/brave/brave-core/blob/0f5f58b7ffd744f884c3869532892182b16897b2/build/commands/lib/util.js#L532-L539

Reply:
Fixed here fe47bcd00a6.